install.rb

```diff
@@ -294,6 +308,16 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    if $operatingsystem == "windows"
+      localedir = File.join(Dir::COMMON_APPDATA, "PuppetLabs", "puppet", "share", "puppet", "locale")
```

Comment:
So I think this should probably go under `Program Files` as static data, rather than living under user data directories, because I would expect the MSI to have all the translations baked in at install-time.

`install.rb` is supposed to roughly mimic how an MSI is installed, so this doesn't quite add up for me... but I might be misunderstanding how this all works?

I also would expect to see a PR to https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md that describes where translations end up.

If you want to add a directory under what is something like `Program Files`, I think you want to mod under https://github.com/Magisus/puppet/blob/848717c34d6accd1f4c2fd3c3c04950db27fc4fe/install.rb#L486 

We should probably chat about this to make sure we're on the same page - I'm coming into this pretty late, and not sure I grok the process yet.